.github/workflows/format.yml

```
@@ -39,39 +39,64 @@ jobs:
         with:
           access_token: ${{ github.token }}
 
-      - name: Set up Python
-        uses: actions/setup-python@v2
-        with:
-          python-version: 3.7
-
       - name: Install dependencies
-        run: sudo apt update && sudo apt -y install clang-tidy-13 cmake g++-10 libomp-13-dev
-        env:
-          DEBIAN_FRONTEND: noninteractive
+        run: sudo apt update && sudo apt -y install clang-format-13 python3
```

```suggestion
        run: sudo apt update && sudo apt -y install clang-format-13 python3
```
I expect this will need updating when #351 merges

---

Yes. I'm waiting for that.